firebase-firestore/src/main/java/com/google/firebase/firestore/VectorValue.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.firestore;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import java.io.Serializable;
+import java.util.Arrays;
+
+/**
+ * Represent a vector type in Firestore documents.
+ * Create an instance with {@link FieldValue#vector(double[])}.
+ */
+public class VectorValue implements Serializable {
```

Comment:
This was a practice I carried over from java-firestore. However, since this is not consistent with other Android value types, I'm removing it, for now.